# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../linux-header/install-missing-netfilter_bridge-headers.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Install ebt_ip6.h and ebt_nflog.h needed to build ebtables

diff -ruN linux-2.6.35.8-orig/include/linux/netfilter_bridge/Kbuild linux-2.6.35.8/include/linux/netfilter_bridge/Kbuild
--- linux-2.6.35.8-orig/include/linux/netfilter_bridge/Kbuild	2010-08-02 00:11:14.000000000 +0200
+++ linux-2.6.35.8/include/linux/netfilter_bridge/Kbuild	2010-12-07 10:57:56.870695940 +0100
@@ -2,11 +2,13 @@
 header-y += ebt_arp.h
 header-y += ebt_arpreply.h
 header-y += ebt_ip.h
+header-y += ebt_ip6.h
 header-y += ebt_limit.h
 header-y += ebt_log.h
 header-y += ebt_mark_m.h
 header-y += ebt_mark_t.h
 header-y += ebt_nat.h
+header-y += ebt_nflog.h
 header-y += ebt_pkttype.h
 header-y += ebt_redirect.h
 header-y += ebt_stp.h
